infra/create-app/main.tf

```diff
@@ -74,3 +74,8 @@ module "ses" {
     source = "../modules/ses"
     environment = local.environment
 }
+
+module "emails" {
+    source = "../modules/emails"
+    environment = local.environment
```

Comment:
this is not used, if you want to use it in the module you'll need to create the variable in a vars.tf file understand module and populate it like so:
```terraform
variable "environment" {
  type = string
}
```